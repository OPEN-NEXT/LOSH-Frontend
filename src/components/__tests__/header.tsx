/* eslint-disable react/display-name */
/* eslint-disable @typescript-eslint/explicit-function-return-type */
// /* eslint-disable no-unused-vars */
import * as React from "react";
import ShallowRenderer from "react-test-renderer/shallow"; // ES6
import Header from "../header";

const renderer = ShallowRenderer.createRenderer();

jest.mock("antd", () => {
	return {
		Layout: {
			Header: () => null
		},
		Row: () => null,
		Col: () => null,
		Menu: () => null,
		Typography: {
			Text: () => null
		}
	};
});

describe("Header", () => {
	it("renders correctly", () => {
		const tree = renderer.render(
			<Header headerLinks={[]} siteTitle="Library of Open Source Hardware" />
		);
		expect(tree).toMatchSnapshot();
	});
});
