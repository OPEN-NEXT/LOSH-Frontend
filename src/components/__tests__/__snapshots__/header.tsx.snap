// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<Header>
  <Row
    align="middle"
    justify="space-between"
  >
    <Col>
      <mockConstructor
        to="/"
      >
        <Text
          strong={true}
          style={
            Object {
              "color": "white",
            }
          }
        >
          Default Starter
        </Text>
      </mockConstructor>
    </Col>
    <Col>
      <Menu
        mode="horizontal"
      />
    </Col>
  </Row>
</Header>
`;
